Algorithms/Dijkstras/c++/PriorityQueue.h

```diff
@@ -0,0 +1 @@
+#pragma once

```

Comment:
Why do we have this file? This could be removed right?

Reply:
It's part of the implementation of the algorithm--I was going to use the priority_queue structure from the containers library, but it didn't have the functionality I was looking for.

Reply:
Can you link to the line where this file is included? I can see `PQueue.h` being included but couldn't find where `PriorityQueue.h` is being used.

Reply:
Oh whoops, sorry about that! Let me fix that--I included the wrong file when I meant to add the test file instead.